### PR TITLE
Docs: Notary v2 and ORAS Artifacts

### PR DESCRIPTION
**Pull request by SteveLasker**
Signed-off-by: Steve Lasker <stevenlasker@hotmail.com>

### CODE REVIEW

**Comment by nate-double-u** [comment]
Deploy preview: https://deploy-preview-28--notarydev.netlify.app/docs/v2/notary-artifacts/
(I like the automated one that Netlify drops in its comment, but I like to have the direct link to the page changed/added)

File: content/en/blog/2021/announcing-notation-alpha1.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+---
+
+We are happy to announce the [alpha 1 release][notation-release] of the [Notary v2 project][notary-v2].
+Notary v2 enables signing of all artifacts (Container Images, Software Bill of Materials, Scan Results) stored in an [OCI Distribution][oci-distribution] based registries.
+A key tenant of Notary v2 enables promotion of signed artifacts within and across registries, including air-gapped and private network environments.
+As artifacts are promoted, users/entities may add new signatures, attesting to the validity of the content for the target environment, enabling a secure supply chain workflow.
+
+Through signing, users choose the artifacts they trust, from the entities they trust, [decoupling location from identity](https://stevelasker.blog/2021/09/24/separating-identity-from-location/).
+
+![](./announcing-notation-alpha1/artifact-promotion.svg)
```
[the remark this answers is not in the document]
Comment (nate-double-u) [reply content/en/blog/2021/announcing-notation-alpha1.md]: sorry, wrong PR. 😅

**Comment by zr-msft** [comment]
@SteveLasker @toddysm is this PR still relevant or should it be closed?

**Comment by zr-msft** [comment]
Closing for now due to inactivity

**Pull request closed without merging**